templates/base24.mustache

```diff
@@ -0,0 +1,61 @@
+# Base24 {{scheme-name}}
+# Scheme author: {{scheme-author}}
+# Template author: Tinted Theming: (https://github.com/tinted-theming)
+
+# default statusbar colors
+set-option -g status-style "fg=#{{base04-hex}},bg=#{{base01-hex}}"
+
+# default window title colors
+set-window-option -g window-status-style "fg=#{{base04-hex}},bg=#{{base01-hex}}"
+
+# active window title colors
+set-window-option -g window-status-current-style "fg=#{{base13-hex}},bg=#{{base01-hex}}"
+
+# pane border
+set-option -g pane-border-style "fg=#{{base01-hex}}"
+set-option -g pane-active-border-style "fg=#{{base05-hex}}"
+
+# message text
+set-option -g message-style "fg=#{{base10-hex}},bg=#{{base12-hex}}"
```

Comment:
Also curious on this change as its the following in base16

```
set-option -g message-style "fg=#{{base06-hex}},bg=#{{base02-hex}}"
```

Reply:
I think I was debugging and left it in, I'll revert.

Reply:
Ah ok cool. I didn't know if it was a case of taking an opinionated approach to messages? A while ago I created a few opinionated 'flavours' for a Qt theme